Rework Migrate section into Import and connect (#4323)
timescale/docs: migrate/index.md

---
title: Connect, import, ingest, and migrate your data to Tiger Cloud
excerpt: In Tiger Cloud, you can easily import individual files, ingest data in bulk, migrate from other databases, or connect directly so that your data from another source is continuously synced 
products: [cloud, self_hosted]
keywords: [import, ingest, connect, migrate, RDS]
tags: [import, ingest, connect, migrate, RDS]
---

import OpenSupportRequest from "versionContent/_partials/_migrate_open_support_request.mdx"

# Import and connect 

In $CLOUD_LONG, you can easily add and sync data to your $SERVICE_SHORT from other sources. This includes:

> - Connecting directly, so that your data from another source is continuously synced to your $SERVICE_SHORT.

Reviewer: ```suggestion
- Connect directly, so data from another source is continuously synced to your $SERVICE_SHORT.
```

> - Importing individual files via $CONSOLE_LONG and the command line.

Reviewer: ```suggestion
- Import individual files using $CONSOLE_LONG or the command line.
```

> - Migrating data from other databases.

Reviewer: ```suggestion
- Migrate data from other databases.
```

> - Ingesting metrics from Telegraf.

Reviewer: ```suggestion
- Ingest metrics from Telegraf.
```

## Connect to $PG or S3

> $CLOUD_LONG provides source connectors for Postgres and S3, which you can use to synchronize all or some of your data to your $SERVICE_LONG in real time. You run the connectors continuously, using your data as a primary database and your $SERVICE_LONG as a logical replica. This enables you

Reviewer: ```suggestion
$CLOUD_LONG provides source connectors for Postgres and S3. You use them to synchronize all or some of your data to your $SERVICE_LONG in real time. You run the connectors continuously, using your data as a primary database and your $SERVICE_LONG as a logical replica. This enables you
```

to leverage $CLOUD_LONG’s real-time analytics capabilities on your replica data.

| Connector options                      |  Downtime requirements |
|----------------------------------------|-----------------------|
| [$PG_CONNECTOR_CAP][livesync-postgres] | None                  |
| [$S3_CONNECTOR_CAP][livesync-s3]       | None                  |

## Import individual files

> You can [import individual files via $CONSOLE_SHORT][import-console], from your local machine or S3. This includes CSV, Parquet, TXT, and MD files. Alternatively, [import data in the terminal][import-terminal].

Reviewer: using, not via

My mistake, using the terminal. Can you check other mentions of in the terminal while you are at it please.

> You can [import individual files via $CONSOLE_SHORT][import-console], from your local machine or S3. This includes CSV, Parquet, TXT, and MD files. Alternatively, [import data in the terminal][import-terminal].

Reviewer: ```suggestion
You can import CSV, Parquet, TXT, and MD files from your local machine or S3 to your $SERVICE_LONG using [$CONSOLE_SHORT][import-console] or [the terminal][import-terminal].
```

Author: This sadly becomes less readable as a single sentence, too many components.
